charts/argocd-notifications/templates/bots/slack/role.yaml

```diff
@@ -15,4 +15,13 @@ rules:
   - watch
   - update
   - patch
+- apiGroups:
+  - ""
+  resources:
+  - secrets
+  - configmaps
+  verbs:
+  - get
+  - list
+  - watch
```

Comment:
Interesting, I know this isn't reflected in https://raw.githubusercontent.com/argoproj-labs/argocd-notifications/stable/manifests/install-bot.yaml, so @JonathanSerafini, I'd like to ask you to follow up after this PR to include a fix for the main argocd-notification project to reflect this, too.

Reply:
https://github.com/argoproj-labs/argocd-notifications/blob/master/manifests/bot/argocd-notifications-bot-role.yaml
```yaml
apiVersion: rbac.authorization.k8s.io/v1
kind: Role
metadata:
  name: argocd-notifications-bot
rules:
- apiGroups:
  - ''
  resourceNames:
  - argocd-notifications-secret
  resources:
  - secrets
  verbs:
  - get
  - list
  - watch
- apiGroups:
  - argoproj.io
  resources:
  - applications
  - appprojects
  verbs:
  - get
  - list
  - watch
  - update
  - patch
```

https://github.com/argoproj-labs/argocd-notifications/blob/master/manifests/install-bot.yaml#L11
```yaml
apiVersion: rbac.authorization.k8s.io/v1
kind: Role
metadata:
  name: argocd-notifications-bot
rules:
- apiGroups:
  - ""
  resourceNames:
  - argocd-notifications-secret
  resources:
  - secrets
  verbs:
  - get
  - list
  - watch
- apiGroups:
  - argoproj.io
  resources:
  - applications
  - appprojects
  verbs:
  - get
  - list
  - watch
  - update
  - patch
```